ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/create-run.tsx

```diff
@@ -55,6 +55,7 @@ import { Label } from '@/components/ui/label';
 import { Separator } from '@/components/ui/separator';
 import { Switch } from '@/components/ui/switch';
 import { useToast } from '@/components/ui/use-toast';
+import { advisors, packageManagers, reportFormats } from './-types';
```

Comment:
@Etsija I think this for example is a good use for relative imports.

Reply:
This was a really elegant way for decluttering. Related to this PR, I have some thoughts about more refactoring: as the create-run page reuses similar form components, they should probably be extracted into their own respective components. I could think of at least `components/form/(input-field.tsx, toggle-field.tsx, checkbox-field.tsx)`.

Reply:
Sounds good! It would be great to have a unified style for forms accross the UI, and shared components would very much help in that.